crates/biome_js_analyze/src/analyzers/nursery/no_default_export.rs

```diff
@@ -0,0 +1,82 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::AnyJsExportClause;
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Disallow `export default`.
+    ///
+    /// In contrast to default exports,
+    /// named exports are discoverable in an Editor: the editor can suggest to import a named export.
+    /// Also, named exports improve consistency by using the same name through a code base.
```

Comment:
The rule is about default exports, so the documentation should explain why this rule bans them. We should not mention them. So we should highlight **why the rule bans them**. The rule is called "noDefaultExports, no `useNamedExports` :)